packages/core/src/store/feed-aggregation-store.ts

```diff
@@ -34,14 +65,20 @@ export class FeedAggregationStore extends ObjectStore<number, StreamID> {
     this.cleanupInterval = cleanupInterval
     this.tasks = new TaskQueue()
     this.#interval = undefined
+    this.find = this.find.bind(this)
+    this.onWrite = new Subject()
   }
 
   findKeys(params?: Partial<StoreSearchParams>): Promise<Array<string>> {
     return this.store.findKeys(params)
   }
 
+  find(params?: Partial<StoreSearchParams>): Promise<Array<IKVStoreFindResult>> {
+    return this.store.find(params)
+  }
+
   async deleteStale(lessThan: number): Promise<number> {
```

Comment:
can we make clear the units being used here?  Maybe just call the field `lessThanMs` assuming it's expecting milliseconds?